base/src/main/java/io/fabric8/launcher/base/JsonUtils.java

```diff
@@ -1,18 +1,35 @@
 package io.fabric8.launcher.base;
 
+import java.io.IOException;
 import java.math.BigDecimal;
 import java.math.BigInteger;
 import java.util.Map;
 
 import javax.json.JsonArrayBuilder;
 import javax.json.JsonObjectBuilder;
 
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
 import static javax.json.Json.createArrayBuilder;
 import static javax.json.Json.createObjectBuilder;
 
 public final class JsonUtils {
 
+    private static final ObjectMapper mapper = new ObjectMapper();
+
     private JsonUtils() {
```

Comment:
Since we use different Json Libraries, maybe naming it Jacksons or JsonMappers?

Reply:
That's an implementation detail, I think we should leave in the JsonUtils class for now and then separate later if needed